user_sessions/views.py

```diff
@@ -43,3 +43,22 @@ class LogoutView(View):
     def get(self, request):
         logout(request)
         return HttpResponseRedirect(reverse('user_sessions:home'))
+
+
+class AddUserView(View):
+
+    form_class = UserRegistrationForm
+    template_name = 'user_sessions/register_user.html'
+
+    def get(self, request):
+        form = self.form_class()
+        return render(request, self.template_name, {'form': form})
+
+    def post(self, request):
+        form = self.form_class(request.POST)
+        if form.is_valid():
+            user = form.save()
+            login(request, user)
```

Comment:
login only sets the session attrs in request. we need to authenticate the given creds to get the authenticated user before login.